### PR TITLE
rename dev server files

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

renames the dev server and webpack config files

## Why?

they're hard to browser right now

### Before

image

### After

image

### CODE REVIEW

File: dotcom-rendering/makefile [hunk 1]
```diff
@@ -152,7 +152,7 @@ reinstall: clear clean-deps install
 validate-build: # private
 	$(call log, "checking bundling")
 	@rm -rf dist
-	@NODE_ENV=production webpack --config ./scripts/webpack/frontend.js
+	@NODE_ENV=production webpack --config ./scripts/frontend/dev-server.js
```
Comment (OllysCoding) [inline dotcom-rendering/makefile:155]: I think this should be going to ./scripts/webpack/webpack.config.js

File: dotcom-rendering/package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
     "cypress:run": "cypress run",
     "cypress:run:e2e": "cypress run --spec 'cypress/integration/e2e/**/*'",
     "cypress:run:mocked": "cypress run --spec 'cypress/integration/mocked/**/*'",
-    "makeBuild": "NODE_ENV=production CI_ENV=github webpack --config ./scripts/webpack/frontend.js"
+    "makeBuild": "NODE_ENV=production CI_ENV=github webpack --config ./scripts/frontend/dev-server.js"
```
Comment (OllysCoding) [inline dotcom-rendering/package.json:31]: I think this should be going to ./scripts/webpack/webpack.config.js

Comment (sndrs) [reply dotcom-rendering/package.json]: yeah i was too vociferous with my pasting. hasty paster

File: dotcom-rendering/scripts/webpack/webpack.config.browser.js [hunk 3]
```diff
@@ -63,7 +63,7 @@ module.exports = ({ isLegacyJS, sessionId }) => ({
 		DEV && friendlyErrorsWebpackPlugin(),
 		DEV &&
 			new GuStatsReportPlugin({
-				buildName: isLegacyJS ? 'client' : 'legacy-client',
+				buildName: isLegacyJS ? 'legacy-client' : 'client',
```
[the remark this answers is not in the document]
Comment (sndrs) [reply dotcom-rendering/scripts/webpack/webpack.config.browser.js:66]: 👍

File: dotcom-rendering/scripts/env/check-node.js [hunk 4]
```diff
@@ -8,8 +8,7 @@ const ensure = require('./ensure');
 	try {
 		const [semver] = await ensure('semver');
 
-		const nodeVersion = process.version.match(/^v(\d+\.\d+\.\d+)/)[1];
-		console.log('join: ' + join(__dirname, '..', '..', '..', '.nvmrc'));
+		const nodeVersion = /^v(\d+\.\d+\.\d+)/.exec(process.version)[1];
```
Comment (sndrs) [inline dotcom-rendering/scripts/env/check-node.js:11]: lint fix updated this to use `exec`

Comment (sndrs) [reply dotcom-rendering/scripts/env/check-node.js:11]: ¯\_(ツ)_/¯

**Review by jamesgorrie — APPROVED** [review]
Lovely

**Review by OllysCoding — APPROVED** [review]
+1 This is a really nice change :) Will for sure make webpack/dev-server changes much easier for future egineers!

**Pull request merged**